srtcore/utilities.h

```diff
@@ -764,6 +827,44 @@ struct MapProxy
     }
 };
 
+inline std::string BufferStamp(const char* mem, size_t size)
+{
+    using namespace std;
+
+    union
+    {
+        char spread[16];
+        uint32_t testin[4];
```

Comment:
'testin' is unused and produces a warning

Reply:
Ok, this was used in some blocked instruction, left as commented-out so that this instr can be re-enabled in future development.